doc/source/io.rst

```diff
@@ -287,7 +287,7 @@ lineterminator : str (length 1), default ``None``
 quotechar : str (length 1)
   The character used to denote the start and end of a quoted item. Quoted items
   can include the delimiter and it will be ignored.
-quoting : int or ``csv.QUOTE_*`` instance, default ``None``
+quoting : int or ``csv.QUOTE_*`` instance, default ``0``
```

Comment:
isn't this still defaulted to ``None``?, meaning don't consider quoting at all? (and NOT ``QUOTE_NONE``?) or are these the same